src/neo/Trie/Trie.cs

```diff
@@ -0,0 +1,20 @@
+
+using System.Collections.Generic;
+
+namespace Neo.Trie
+{
+    public interface ITrie
+    {
+        bool TryGet(byte[] path, out byte[] value);
+
+        bool TryPut(byte[] path, byte[] value);
+
+        bool TryDelete(byte[] path);
+
+        byte[] GetRoot();
+
+        Dictionary<byte[], byte[]> GetProof(byte[] Key);
+
+        void Commit();
+    }
+}
```

Comment:
A blank line should be added in the end of the file, otherwise the dotnet-format won't pass.